splash/qtrender_image.py

```diff
@@ -457,13 +465,13 @@ def crop(self, rect):
         self.img = self.img.crop((left, top, right, bottom))
 
     def to_png(self, complevel=defaults.PNG_COMPRESSION_LEVEL):
-        buf = StringIO()
+        buf = BytesIO()
```

Comment:
Could you please check that this change doesn't make performance worse in Python 2.x?